index.html

```diff
@@ -507,7 +512,90 @@ <h3>StatusList2021Credential</h3>
+                MUST be processed as `1`.  <code>size</code> MUST be an integer greater than zero.
+                if <code>size</code> is provided, and is greater than `1`, then the property
+                <code>credentialStatus.statusMessages</code> MUST be present, and the number of
+                status messages must equal the number of possible values.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                credentialSubject.statusMessages
+              </td>
+              <td>
+                The <code>statusMessages</code> property MUST be an array, and if present
+                the length of the array must equal the number of possible status states
+                indicated by <code>size</code>. <code>statusMessages</code> MAY be present if
+                <code>size</code> is `1`. <code>statusMessages</code> MUST be present if
+                <code>size</code> is greater than `1`.  If not present, the message value
+                associated with the bit value of <code>0</code> is "unset" and the bit
+                value of <code>1</code> is "set".
+                Elements in the <code>statusMessages</code> array if present, MUST contain at 
+                minimum two properties:
+                <ul>
+                  <li><code>status</code> being a string of the hex value of the status</li>
+                  <li><code>value</code> a string containing the associated message</li>
+                </ul>
+                Implementers MAY add additional values to objects in the <code>statusMessages</code>
+                array.
+                Implementers MAY add a boolean value to elements in the <code>statusMessages</code>
+                named <code>verify</code> which adds an instruction to verifiers where if the
+                value of <code>verify</code> is <code>true</code> the verifier SHOULD not
+                use the status to reject verification of the verifiable credential. If the
+                value of <code>verify</code> is <code>false</code>, then a verifier SHOULD
+                NOT verify the verifiable credential.
+                If <code>verify</code> is present, then the values of `0` and `1` should 
+                be processed identically to the values of `0` and `1` in a status list with a
+                <code>statusPurpose<code> of <code>suspension</code>.
+                Implementers MAY use the string value of <code>undefined</code> in the value
+                to indicate that a status corresponding is not defined for the associated
+                status value, but that it may be definied in the future.
+                Rules for how to handle various status messages are outside the scope of
+                normative reuqirements in this document, but it is assumed that implementers
+                SHOULD and will document rules for processing various status codes.
```

Comment:
```suggestion
                to indicate that a corresponding status is not defined for the associated
                status value, but that it may be defined in the future.
                Rules for how to handle various status messages are outside the scope of
                normative requirements in this document, but it is assumed that implementers
                will document rules for processing various status codes.
```